operations/mimir/redis.libsonnet

```diff
@@ -0,0 +1,57 @@
+local redis = import 'redis/redis.libsonnet';
+
+redis {
```

Comment:
We need to be setting max memory for all the Redis caches as well as using `allkeys-lru` for eviction. We also need to disable all persistence.